homework-2/src/app/components/feature/shows/ShowReviewSection .tsx

```diff
@@ -0,0 +1,64 @@
+"use client";
+
+import { useState } from "react";
+import { Box, Flex } from "@chakra-ui/react";
+import { ReviewList } from "../reviews/ReviewList";
+import { ReviewForm } from "./ReviewForm";
+import { useEffect } from "react";
+import { IReviewItemProps } from "../reviews/ReviewItem";
+
+export interface IReview {
+  email: string;
+  avatar: string;
+  rating: number;
+  comment: string;
+}
+
+const saveToLocalStorage = (reviews: IReview[]) => {
+  localStorage.setItem("reviews", JSON.stringify(reviews));
+};
+
+const loadFromLocalStorage = () => {
+  const reviewsString = localStorage.getItem("reviews");
+  if (reviewsString) {
+    return JSON.parse(reviewsString);
+  }
+  return [];
+};
+export const ShowReviewSection = () => {
+  const [reviews, setReviews] = useState<IReview[]>([]);
```

Comment:
`IReview[]` is the same as `Array<IReview>`. In Infinum, we prefer `Array<IReview>` just to be consistent.